pkg/registry/global_registry.go

```diff
@@ -0,0 +1,25 @@
+package registry
```

Comment:
why do we need a new file? Can this be merged with registry.go as there is only one global map?

Reply:
you're right. I was thinking that it is just a temporary file that further changes to global registry can be made in another PR. Merging it with registry.go is better here. Thanks!